s2_jv_electric_boogaloo/mainGUI.cpp

```diff
@@ -26,14 +30,25 @@ void MainGUI::gridUpdate(char grid[GRID_X][GRID_Y])
 
 void MainGUI::changePage(int page) {
     screen1->hide();
+    screen2->hide();
     qWarning() << "page change";
 
     if (page == 1) {
+        screen1->hide();
+        screen2->hide();
         thread = QThread::create([this] {
             Gameloop gameloop;
             connect(&gameloop, &Gameloop::gridUpdate, this, &MainGUI::gridUpdate);
             gameloop.mainLoop();
             });
         thread->start();
     }
+    if (page == 2) {
```

Comment:
else if, sert a rien d'évaluer si page == 1
```suggestion
    else if (page == 2) {
```